.gitignore

```diff
@@ -2,3 +2,4 @@
 **/*.rs.bk
 /Cargo.lock
 .cargo
+.vscode
```

Comment:
Minor nit: These should probably go into your own `$HOME/.gitignore` so that we don’t need to have lines for each and every editor.

Reply:
Minor minor nit: I believe you mean `$HOME/.gitconfig`.

Reply:
Ah, I have this in my .gitconfig:

```
[core]
        excludesfile = ~/.gitignore
```

Reply:
Ah yes. Good point! Will remove this.